hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/LinuxContainerExecutor.java

```diff
@@ -525,6 +580,23 @@ public int launchContainer(ContainerStartContext ctx) throws IOException {
         logOutput(diagnostics);
         container.handle(new ContainerDiagnosticsUpdateEvent(containerId,
             diagnostics));
+        if (exitCode == LinuxContainerExecutorExitCode.
```

Comment:
I get enum types cannot be instantiated. I could create a function that returns the appropriate enum for an int value, but would not that be an overkill here?